### ISSUE TITLE
sideNav has no keyboard (or screen reader) support 

### ISSUE THREAD

**Issue by cptvitamin** [Dogfalo/materialize#925]
Currently, there is no way to move focus to, or activate the sideNav widget with the keyboard.
Also, there are no aria-* attributes to indicate the current state of the menu (aria-expanded) or it's properties (aria-controls).
There is also no focus management upon activation (focus should move to the sidebar region upon open and back to menu control upon close)

**Comment by SamWillson** [comment]
A temporary solution to all those with the same problem can be, to use a external script / plugin.
For Screen readers you can use the Accessibility API and scripting events for keyboard isn't really that hard. Along with jQuery you can attach the events to sideNav or any component in general. This can serve as a temporary solution until devs add some feature to library.
